awx/api/serializers.py

```diff
@@ -1715,6 +1715,7 @@ def get_related(self, obj):
             res['organization'] = self.reverse('api:organization_detail', kwargs={'pk': obj.organization.pk})
         if obj.kind == 'constructed':
             res['source_inventories'] = self.reverse('api:inventory_source_inventories', kwargs={'pk': obj.pk})
+            res['url'] = self.reverse('api:constructed_inventory_detail', kwargs={'pk': obj.pk})
```

Comment:
is `url` descriptive enough?

Reply:
it's a pre-existing field, I'm swapping the `/api/v2/inventories/42/` link with `/api/v2/constructed_inventories/42/` link. Somewhat opinionated, either would work, but I figured this is what we're going for.